### DISCUSSION TITLE
Steps needed for non-techie

### DISCUSSION THREAD

**Discussion by wareson** [Chia-Network/chia-blockchain#1374] [Q&A]
Hello all!  Keen to get involved, host a plot, and get farming!  Have a good spec of laptop but read guidance about using different drive to the one housing the OS plus don't have enough space.  What are some good options on external hard drives that I could hook up for the purpose?  (replies with links would be amazing - UK-based but sure I could find vendors so any and all appreciated)

**Comment by hoffmang9** [comment]
This is a good place to start - https://github.com/Chia-Network/chia-blockchain/wiki/Reference-Plotting-Hardware

**Reply by wareson** [reply]
Thank you. Seems to have good info for desktops. Before embarking on a desktop build, I'm trying to work out if my gaming laptop with external hard drive could be 'useful' from TiB/day perspective and if so what external hard drive I'd have to hook up?

**Reply by dorkmo** [reply]
you'll have to read the spec's but just look for an external ssd with fast usb, and an external hard disk with lots of TB. its not going to be as fast as plotting on an M.2 board connected to the motherboard, but laptops typically only have one M.2 slot and its for the boot drive, and you dont want to plot on the boot drive.

**Reply by mmmarcus** [reply]
@dorkmo can you please give an example of those two products so I could try it out with my laptop?
If I understand it right I would need an external NVMe SSD drive with at least 1 TB, and then another external drive with loads of TB to transfer it to?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
